src/main/java/com/stripe/param/PaymentIntentConfirmParams.java

```diff
@@ -1517,7 +1447,7 @@ public Builder putAllExtraParam(Map<String, Object> map) {
     public static class BillingDetails {
       /** Billing address. */
       @SerializedName("address")
-      Address address;
+      Object address;
```

Comment:
I don't think I noticed this change in the changelog.

Reply:
yeah I don't usually document when we support unsetting hashes